upf/upf_pfcp_server.c

```diff
@@ -1052,19 +1057,15 @@ u32 upf_pfcp_server_start_timer (u8 type, u32 id, u32 seconds)
 			      interval);
 }
 
-void upf_pfcp_server_deferred_release_association (u32 node)
+void upf_pfcp_server_deferred_free_msgs (u32 node, u32 sidx)
```

Comment:
does it really make sense to combine the node and sidx release into one function? There is no caller that releases both at the same time.

Reply:
Okay, will make it 2 functions